xsd/netex_part_1/part1_networkDescription/netex_line_version.xsd

```diff
@@ -756,6 +761,11 @@ Rail transport, Roads and Road transport
 				</xsd:annotation>
 			</xsd:element>
 			<xsd:group ref="DestinationDisplayNameGroup"/>
+			<xsd:element name="Presentation" type="PresentationStructure" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Presentation values to use when rendering DESTINATION DISPLAY VARIANT, such as a colour.</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
```

Comment:
This part of the pull request has not yet seen in master. @nick-knowles

Reply:
I'll suggest here to approach it as new pull request. That prevents a mess.

Reply:
This part is not in master yet. @Aurige : how to proceed.